web/styles/settings.css

```diff
@@ -1463,9 +1466,8 @@ label.preferences-radio-choice-label {
         .exit-sign {
             float: right;
             position: relative;
-            top: 1px;
```

Comment:
This isn't necessary anymore because the contents are vertically centered with flex.